.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
-  - lts/dubnium
+  - 8
+  - 10
+  - 12
```

Comment:
Testing across node versions is a welcome addition. :+1: 
Please use the LTS codenames `lts/carbon`, `lts/dubnium`, and `lts/erbium` over the number versions.
For consistency with other unified/remark projects.